File: problems/algorithms/string-util/js/response/string-util.print.js
import { formatter } from "./string-util.js";

// String Util

// changing case for text to uppercase
console.log(formatter("lorem ipsum dolor", "uppercase"));
console.log("LOREM IPSUM DOLOR");

// change case for text to lowercase
console.log(formatter("LOREM IPSUM DOLOR", "lowercase"));
console.log("lorem ipsum dolor");

// change case for text to camelcase
console.log(formatter("LOREM IPSUM DOLOR", "camelcase"));
console.log("Lorem Ipsum Dolor");

// change case for text to snakecase
console.log(formatter("lorem ipsum dolor", "snakecase"));
console.log("lorem_ipsum_dolor");

// reversing text
console.log(formatter("lorem ipsum dolor", "reverse"));
console.log("rolod muspi merol");

// counting chars
console.log(formatter("lorem\nipsum dolor", "countchar"));
console.log(17);

// counting words
console.log(formatter("lorem\nipsum dolor", "countword"));
console.log(3);

// counting lines
console.log(formatter("lorem\nipsum dolor", "countline"));
console.log(2);

File: src/pages/problems/string-util/_codes/ruby/response/print/string_util.print.rb
require_relative "../src/string_util"

# String Util

# changing case for text to uppercase
p formatter('lorem ipsum dolor', 'uppercase')
p 'LOREM IPSUM DOLOR'

# change case for text to lowercase
p formatter('LOREM IPSUM DOLOR', 'lowercase')
p 'lorem ipsum dolor'

# change case for text to camelcase
p formatter('LOREM IPSUM DOLOR', 'camelcase')
p 'Lorem Ipsum Dolor'

# change case for text to snakecase
p formatter('lorem ipsum dolor', 'snakecase')
p 'lorem_ipsum_dolor'

# reversing text
p formatter('lorem ipsum dolor', 'reverse')
p 'rolod muspi merol'

# counting chars
p formatter('lorem\nipsum dolor', 'countchar')
p 17

# counting words
p formatter('lorem\nipsum dolor', 'countword')
p 3

# counting lines
p formatter('lorem\nipsum dolor', 'countline')
p 2